index.js

```diff
@@ -135,7 +141,7 @@ File.prototype.inspect = function() {
   var inspect = [];
 
   // Use relative path if possible
-  var filePath = (this.base && this.path) ? this.relative : this.path;
+  var filePath = this.path ? this.relative : this.path;
```

Comment:
My bad. I've just removed the now unnecessary `this.base` check without re-evaluating the ternary. That's how you arrive at ternaries like `true ? foo : bar` :) Gonna change it to `null`.